### DISCUSSION TITLE
is there a way to find out who the ghost girl is targeting?

### DISCUSSION THREAD

**Discussion by zx3800** [winstxnhdw/lc-hax#267] [General]
it will make trolling with the possession feature a bit easier

**Comment by joep26020** [comment]
> it will make trolling with the possession feature a bit easier

I suppose whoever she is closest to but the question is, is she really targeting the person closest to her? Usually not.

**Comment by winstxnhdw** [comment]
If you have ESP turned on, you can see the Ghost Girl’s box.
